app/counters/subject_workflow_counter.rb

```diff
@@ -11,6 +11,10 @@ def classifications
       .where(workflow: swc.workflow_id)
       .joins("INNER JOIN classification_subjects cs ON cs.classification_id = classifications.id")
       .where("cs.subject_id = ?", swc.subject_id)
+      .where("gold_standard IS NOT TRUE")
+      .where.not("metadata ? 'seen_before'")
```

Comment:
Does this need an index or can PG reduce the scope plenty with the other ones so that a scan isn't an issue?

Reply:
see the PR comment for details on the query plan, noting it's not a scan but an index with filter. For ref, there is an index on workflow_id but the query planner chooses not to use it due to the cost of combining indexes and because join (index query) returns a list classification ids that match the id index. 

It is normally cheaper to walk the selective index (join in this case) looking up rows and apply's filters to it than merging index scans, especially if one is less selective. I believe we would need a compound id, workflow_id, gs, metadata index here to support this particular query (though i'm not sure the metadata one would work, might need promotion to a col). If we had that index it would be a straight index query and defs, faster. That said in basic testing it wasn't an issue and i'm happy to do some better test query profiling on this if you want.